### ISSUE TITLE
Rainbow theme causes @ tags and links to be indistinguishable from normal text

### ISSUE THREAD

**Issue by TrainDoctor** [eramdam/BetterTweetDeck#802]
**Describe the bug**
Rainbow theme causes links to blend into normal text making them hard to distinguish.

**To Reproduce**

0. Open Firefox and open Tweetdeck.
1. Go to "BTD Settings"
2. Click on the "Theme" tab.
3. Select the Rainbow theme and hit "Save"
4. Reload your active Tweetdeck tab(s).
5. Look for in-line links and @ tags etc.
6. Observe the white text in place of highlighted text of another color.

**Expected behavior**
I would expect the color of links to be distinct from the white text that normally makes up the tweet.
Having the link color be white would be fine, if the other text was a different color.

**Screenshots**
Original (Blue Whale Theme)
![Screenshot 2022-03-18 at 12-31-08 TweetDeck](https://user-images.githubusercontent.com/11465594/159072143-044e14f8-7a2d-46a7-a71b-616cef387492.png)
Rainbow Theme
![Screenshot 2022-03-18 at 12-29-29 TweetDeck](https://user-images.githubusercontent.com/11465594/159072203-b3033a6a-72f5-478d-8ce9-065655a7a6fb.png)

```
System:
  Kernel: 5.16.14-arch1-1 x86_64 bits: 64
    Desktop: KDE Plasma 5.24.3 Distro: Arch Linux
  Firefox
    version: 98.0.1
```

**Additional context**
If this is in fact intentional then I will close the bug report if instructed to.
I would highly suggest this be changed even if it is intended behavior.

**Comment by eramdam** [comment]
The "rainbow" theme is meant to allow you to choose a custom color when selected. The next release will (hopefully) make that clearer by showing the color picker right below the form 😅

https://user-images.githubusercontent.com/1409924/159077519-a8d19cbb-b2a3-4ab2-aa71-daf78f7538f9.mp4

**Comment by TrainDoctor** [comment]
> 

Oh gotcha well I can report that there might be a bug in there somewhere as my color picker defaults to the bottom left which would be black seemingly but it is in fact white. Dunno if that's important enough to keep this thread open. Cheers for the quick response!

**Comment by eramdam** [comment]
> Oh gotcha well I can report that there might be a bug in there somewhere as my color picker defaults to the bottom left which would be black seemingly but it is in fact white. Dunno if that's important enough to keep this thread open. Cheers for the quick response!

Yeah that's another issue 😅 The next release will also make sure you don't end up in that situation 😁

**Comment by TrainDoctor** [comment]
> > Oh gotcha well I can report that there might be a bug in there somewhere as my color picker defaults to the bottom left which would be black seemingly but it is in fact white. Dunno if that's important enough to keep this thread open. Cheers for the quick response!
> 
> Yeah that's another issue sweat_smile The next release will also make sure you don't end up in that situation grin

Well that's good to know. You all are doing excellent work doing what twitter should be doing for their own software. Cheers!

**Comment by eramdam** [comment]
Should be fixed in 4.8.5! https://twitter.com/BetterTDeck/status/1505859945639682048
